### PR TITLE
Reverse an array is added

### PR DESCRIPTION
**Pull request by ManavJuneja**

### CODE REVIEW

File: Reverse,.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include<iostream>
+using namespace std;
+/*void swap(int arr[start],int arr[end]){
```
Comment (Saikat2407) [inline Reverse,.cpp:3]: remove all the commented code

File: Reverse,.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#include<iostream>
```
Comment (Saikat2407) [inline Reverse,.cpp:1]: on the first line add the question link.

**Review by Saikat2407 — CHANGES_REQUESTED** [review]
change this file directory to leetcode directory!

File: Reverse,.cpp (hunk 1 above)
Comment (javamultiplex) [reply Reverse,.cpp:3]: > change this file directory to leetcode directory!

This problem is not present in leetcode but present in CodeChef so adding in CodeChef directory

**Pull request merged**